plugins/systemd/systemd_units

```diff
@@ -38,10 +38,12 @@ An example configuration might be something like this:
  env.inactive_critical 20
  env.exclude boring
  env.inactive_exclude sleepy
+ env.silence_active_extinfo 1
 
 In the example above, we've overridden the default warning and critical levels for failed units, added warning
 and critical levels for inactive units, then filtered out boring units from all results and filtered out sleepy
-units from results for the inactive state.
+units from results for the inactive state. In addition to that, only more extensive info of non-active units
```

Comment:
Please mention the default value of `silence_active_extinfo`.

Reply:
Done.